tedana/workflows/tedana.py

```diff
@@ -445,7 +445,7 @@ def tedana_workflow(
         os.mkdir(out_dir)
 
     # boilerplate
-    basename = "report"
+    basename = f"{prefix + '_' if (prefix != '' and not prefix.endswith('_')) else prefix}report"
```

Comment:
Should you also include the prefix for the logfile name on line 461?
That is
461: `basename = f"{prefix + '_' if (prefix != '' and not prefix.endswith('_')) else prefix}tedana_`